### PR TITLE
adding localise bundle recipe

### PR DESCRIPTION
**Pull request by mradhi**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: cosavostra/localise-bundle/1.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "CosaVostra\\LocaliseBundle\\CosaVostraLocaliseBundle": ["all"]
+    },
+    "env": {
+        "LOCALISE_EXPORT_KEY": "localise_api_key_to_replace"
```
Comment (Nyholm) [inline cosavostra/localise-bundle/1.2/manifest.json:6]: I dont see that this is used by your bundle's configuration.

File: cosavostra/localise-bundle/1.2/manifest.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    },
+    "gitignore": [
+        "!/translations/.gitignore",
+        "!/translations/.gitkeep",
```
Comment (Nyholm) [inline cosavostra/localise-bundle/1.2/manifest.json:10]: `.gitkeep` is not a standard file name.

File: cosavostra/localise-bundle/1.2/manifest.json (hunk 1 above)
Comment (mradhi) [reply cosavostra/localise-bundle/1.2/manifest.json:6]: @Nyholm Check https://github.com/mradhi/localise-bundle/blob/master/Resources/config/services.xml, I used this env variable in `CosaVostra\LocaliseBundle\Http\Client` service

Comment (Nyholm) [reply cosavostra/localise-bundle/1.2/manifest.json:6]: You are referencing me to your service definition. 
Your bundle dont even have configuration: https://github.com/mradhi/localise-bundle/tree/master/DependencyInjection

The issue with your current setup is that if a user is missing the environment variable, they will not get an exception until runtime.
I suggest you create some configuration to your bundle and make sure to use that env var there. If the env var is missing then, the user will get an error at build time, which is much more preferred.

Comment (Nyholm) [reply cosavostra/localise-bundle/1.2/manifest.json:6]: Friendly ping

**Review by Nyholm — CHANGES_REQUESTED** [review]

**Comment by Nyholm** [comment]
Feel free to make a new PR if you are still interested to finish this.

**Pull request closed without merging**